ingest_api/runtime/src/schema_helpers.py

```diff
@@ -41,10 +41,10 @@ def check_extent(cls, v):
 
 class TemporalExtent(BaseModel):
     startdate: datetime
-    enddate: datetime
+    enddate: Union[datetime, None]
 
     @root_validator
     def check_dates(cls, v):
-        if v["startdate"] >= v["enddate"]:
+        if (v["enddate"] is not None) and (v["startdate"] >= v["enddate"]):
```

Comment:
Since [both start and end can be open in the spec](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#temporal-extent-object) (although not recommended) should we just allow it here, too?

Reply:
yeah good point, could have null for both start and end. I'll make a change